spec/rails_helper.rb

```diff
@@ -60,6 +60,7 @@
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
 
+  config.include Shoulda::Matchers::Independent
```

Comment:
The newer shoulda-matchers versions inject specific matchers for specific types of tests, but it only works from within the context of a real Rails app apparently 🤷